lang/po/new/cs.po

```diff
@@ -1163,7 +1160,7 @@ msgstr "Vpravo [um]"
 # MSG_RPI_PORT c=13
 #: messages.c:142
 msgid "RPi port"
-msgstr ""
+msgstr "RPi port"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.